packages/react-relay/modern/__tests__/ReactRelayQueryRenderer-test.js

```diff
@@ -604,8 +618,8 @@ describe('ReactRelayQueryRenderer', () => {
             __id: '4',
           },
         },
-        retry: null,
-      }).toBeRendered;
+        retry: jasmine.any(Function),
+      }).toBeRendered();
```

Comment:
Just want to call out this change here. It's the only test that was changed other than adding `expect.assertions` and `await`'s. It looked like an error that `toBeRendered` was not invoked and `retry` is always passed to `render` when props are successfully fetched.